appveyor.yml

```diff
@@ -91,16 +91,20 @@ for:
   # scripts to run after tests
   after_test:
   - ps: |
+      & pwd
       & Setup\BuildInstallers.cmd
-      if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode) }
       & Setup\Set-Portable.ps1
-      if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode) }
       & Setup\MakePortableArchive.cmd Release $env:APPVEYOR_BUILD_VERSION
-      if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode) }
-
+      & tree /F /A > tree.txt
+      Get-ChildItem .\tree.txt | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }
```

Comment:
Don't think we need to publish this either...
https://ci.appveyor.com/project/gitextensions/gitextensions/builds/21627337/artifacts

Reply:
That is a tool to diagnose can't find x file.  It only shows in non release builds.  At least comment it out so that i't a simple uncomment to get it back.